### DISCUSSION TITLE
Sharp Moves of Crazyflie

### DISCUSSION THREAD

**Discussion by emreenes1** [bitcraze/discussions#1381] [Q&A]
I am using Crazyflie 2.1 drone with firmware version 2024.2. when I write python script with motion commander library. I use commands like forward or back, but the drone does not stop and continues to drift when an obstacle appears in front of it while traveling 0.3m. In my work, the drone needs to stop suddenly when it sees an obstacle. But I can't do this using pyton script in the Bitcraze site and I have all the sensors which are multiranger deck and flowdeck. Can you help me?
Code and Video: https://www.bitcraze.io/documentation/tutorials/getting-started-with-stem-drone-bundle/

**Comment by gemenerik** [comment]
Check out [the wall following example](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/multiranger/multiranger_wall_following.py). You can use it as an example to get out multiranger measurements and affect flight behavior based on those measurements.
